ArkhamOverlay/Services/UiEventBus.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ArkhamOverlay.Common.Services;
+using System;
+using System.Windows;
+using System.Windows.Threading;
+
+namespace ArkhamOverlay.Services {
+    public class UiEventBus : EventBus {
+        protected override void DoInvoke<T>(Action<T> action, T eventToPublish) {
```

Comment:
I'm a little concerned about making every event in the main app go through this. I think it's better for the individual actions to know whether they need to do this or not.

Reply:
Discussed offline. Keep an eye on this as more events are added.